src/yeahml/build/layers/recurrent.py

```diff
@@ -1,184 +0,0 @@
-import tensorflow as tf
```

Comment:
forgot about all this nastiness -- yeah let's delete, thanks. In the future, we may have to come up with our own logic for creating recurrent "layers" (separating the building of a cell vs layer), but let's do that in the future ha